Created documentation for security plugin reloadcerts endpoints (#744)
opensearch-project/documentation-website: _security-plugin/access-control/api.md

  ]
}
```
### Reload certificates
Introduced 1.0
{: .label .label-purple }

> Rest API action to reload SSL certificates. It can be used to reload SSL certificates that are about to expire without restarting OpenSearch node. 

Reviewer: Let's follow the format of the other APIs in the file. 
```suggestion
Reloads SSL certificates that are about to expire without restarting the OpenSearch node. 
```

> This API assumes that new certificates are in the same location specified by the security configurations in opensearch.yml. To keep sensitive certificate reload secure, this API only allows hot reload with certificates issued by the same Issuer and Subject DN and SAN with expiry dates after the current one.

Reviewer: ```suggestion
This call assumes that new certificates are in the same location specified by the security configurations in opensearch.yml. To keep sensitive certificate reloads secure, this call only allows hot reload with certificates issued by the same issuer and subject DN and SAN with expiry dates after the current certificate.
```
